Circular_Linked_List/Circular_Linked_List.java

```diff
@@ -0,0 +1,135 @@
+class node{
+    int data;
+    node next;
+    
+    node(int data, node next){
+        this.data=data;
+        this.next=next;
+    }
+}
+
+class cirLinkedList{
+    node head;
+    node tail; //tail is the node that points to the head. created to complete some operations like append in O(1) instead of O(n).
```

Comment:
Add a comment before the line that you want to explain, and add a blank line before the comment line.
If the comment is more than 100 characters, split it into multiple lines.